sequencer/dbmanager.go

```diff
@@ -58,8 +54,7 @@ func (d *dbManager) Start() {
 	go d.loadFromPool()
 	go func() {
 		for {
-			// TODO: Move this to a config parameter
-			time.Sleep(wait * time.Second)
+			time.Sleep(d.cfg.PoolRetrievalInterval.Duration)
```

Comment:
PoolRetrieveInterval is used to polling for pending txs in the pool. In this go func we are detecting a L2Reorg, then we need to create an additional parameter to define this polling interval or let it hard-coded to the previous value (wait = 5 seconds)

Reply:
Yes, this MUST be a different configuration parameter.